### PR TITLE
allow server messages to safely word wrap

### PR DESCRIPTION
**Pull request by ianopolous**
Screenshot_2020-08-05 Peergos

### CODE REVIEW

File: src/components/feedback/feedback.html [hunk 1]
```diff
@@ -44,7 +44,10 @@ <h3>You can tell us here how we can improve, or you can chat with us on <a href=
                                     <pre style="background-color: #ffffff;">{{message.contents}}</pre>
```
Comment (kevodwyer) [inline src/components/feedback/feedback.html:44]: What about here?

Comment (ianopolous) [reply src/components/feedback/feedback.html]: Fixed

File: src/components/feedback/feedback.html [hunk 2]
```diff
@@ -40,8 +40,11 @@ <h3>You can tell us here how we can improve, or you can chat with us on <a href=
                         </tr>
                         <tr v-if="message.visible">
                             <td>
-                                <div v-if="message.from == 'FromUser'">
-                                    <pre style="background-color: #ffffff;">{{message.contents}}</pre>
+                                <div v-if="message.from == 'FromUser'" style="background-color: #ffffff;">
```
Comment (kevodwyer) [inline src/components/feedback/feedback.html:43]: A simplification would be to move the if statement into the style (I wish i did that when i originally wrote it).

Comment (ianopolous) [reply src/components/feedback/feedback.html:43]: what is the syntax for that?

**Pull request merged**